### DISCUSSION TITLE
How to achieve the same workspace as idea? One project, one window.

### DISCUSSION THREAD

**Discussion by bobcbui** [apache/netbeans#6456] [Q&A]
😀

**Comment by neilcsmith-net** [comment]
You might want to look at using project groups.  It's not exactly what you're asking for, but NetBeans has a different workflow and UI.

**Reply by Chris2011** [reply]
It could be possible with a workaround. Afaik you only can have one instance per user folder so we need an action to open a project in a new instance with opening netbeans with a different user dir and if closing, delete the userdir which is not the default one. What do you think? I also wanted this feature VS Code can also handle this. Biggest problem is, NetBeans needs a bit of time to some itselfs up so it is not that fast of opening the project immediately.

Another approach that I thought is just open a JFrame with all the components of NetBeans MainWindow and open the project there. First thing will be easy, second not so I guess.
